.gitignore

```diff
@@ -16,8 +16,10 @@ pids
 **/.DS_Store
 
 # CocoaPods
-/apps/macOS/Pods/*
-/apps/macOS/Podfile.lock
+/apps/macos/Pods/*
+/apps/macos/Podfile.lock
+/apps/ios/ios/Pods/*
+/apps/ios/ios/Podfile.lock
```

Comment:
Many projects I've seen check in their Podfile.lock much like a package-lock.json so that the versioning stays consistent between builds and between machines unless you intentionally make an update call. Is it a react native pattern we're following to ignore the lock files?

Reply:
Not that I know of. Just a general rule to not check in generated files, but agree that it'll be more reliable checking this in

Reply:
Yeah I think package manager lock files are usually an exception to that rule.